framework/source/class/qx/bom/webfonts/Validator.js

```diff
@@ -123,6 +128,14 @@ qx.Class.define("qx.bom.webfonts.Validator", {
       apply : "_applyFontFamily"
     },
 
+    /**
+     * Comparison string used to check whether the font has loaded or not.
+     */
+    comparisonString :
+    {
+      nullable : true,
+      init : null
```

Comment:
Is there a reason not to just eliminate COMPARISON_STRING and set the init value here to what that string was? Then at line 310, the code needn't look at this as a separate default.